Adding documentation for ELM questions v1 (#18356)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/accesspackageanswer.md

---
title: "accessPackageAnswer resource type"

> description:  "The complex type for answers to an accessPackageQuestion that are stored on an accessPackageAssignmentRequest."

Reviewer: ```suggestion
description:  "Represents answers to an accessPackageQuestion that are stored on an accessPackageAssignmentRequest."
```

author: "saylee-dharne-msft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessPackageAnswer resource type

Namespace: microsoft.graph

> The complex type of a requestor's provided answer to an [accessPackageQuestion](../resources/accesspackagequestion.md). The actual answer will be a subtype of this complex type, either [accessPackageAnswerString](../resources/accesspackageanswerstring.md) or [accessPackageAnswerChoice](../resources/accesspackageanswerchoice.md). These answers will be stored on an [accessPackageAssignmentRequest](../resources/accesspackageassignmentrequest.md).

Reviewer: ```suggestion
Represents the answer a requestor provides to an [accessPackageQuestion](../resources/accesspackagequestion.md). The actual answer will be a subtype of this complex type, either [accessPackageAnswerString](../resources/accesspackageanswerstring.md) or [accessPackageAnswerChoice](../resources/accesspackageanswerchoice.md). These answers will be stored on an [accessPackageAssignmentRequest](../resources/accesspackageassignmentrequest.md) object.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |answeredQuestion|[accessPackageQuestion](../resources/accesspackagequestion.md)|The question the answer is for. Required and Read-only.|

Reviewer 2: Not critical, but for a data type within the assignmentRequest, the reader might not know what 'required and read-only' means.  E.g., it's the customer's app that creates a assignmentrequest and so they supply the answeredRequest in their POST payload.

> |displayValue|String|The localized display values shown to the requestor and approvers. |

Reviewer 2: The description has values (plural) but the property name is value (singular).
